app/src/main/java/com/oracle/javafx/scenebuilder/app/menubar/MenuBarController.java

```diff
@@ -1269,22 +1274,36 @@ private void handleOnActionMenu(MenuItem i) {
      */
     
     final static double[] scalingTable = {0.25, 0.50, 0.75, 1.00, 1.50, 2.0, 4.0};
-    
+
     private void updateZoomMenu() {
         final double[] scalingTable = {0.25, 0.50, 0.75, 1.00, 1.50, 2.0, 4.0};
 
+        zoomInController = new ZoomInActionController();
         final MenuItem zoomInMenuItem = new MenuItem(I18N.getString("menu.title.zoom.in"));
-        zoomInMenuItem.setUserData(new ZoomInActionController());
-        zoomInMenuItem.setAccelerator(new KeyCharacterCombination("+", modifier)); //NOI18N
+        zoomInMenuItem.setUserData(zoomInController);
         zoomMenu.getItems().add(zoomInMenuItem);
-        
+
+        zoomOutController = new ZoomOutActionController();
         final MenuItem zoomOutMenuItem = new MenuItem(I18N.getString("menu.title.zoom.out"));
-        zoomOutMenuItem.setUserData(new ZoomOutActionController());
-        zoomOutMenuItem.setAccelerator(new KeyCharacterCombination("/", modifier));  //NOI18N
+        zoomOutMenuItem.setUserData(zoomOutController);
         zoomMenu.getItems().add(zoomOutMenuItem);
-        
+        if (EditorPlatform.IS_MAC) {
+            zoomInMenuItem.setAccelerator(new KeyCharacterCombination("+", modifier)); // NOI18N
+            zoomOutMenuItem.setAccelerator(new KeyCharacterCombination("-", modifier)); // NOI18N
+            // Neither KeyCode.MINUS or KeyCode.SUBTRACT seem to work on macOS.
+            // No chance to test on keyboard with num key pad yet.
```

Comment:
This is definitely a strange behavior 🤔

Reply:
Yes, I can reproduce this on mac.

Reply:
May be this requires a different approach depending on the keyboard type. Are you using a separate keyboard or numpad? I've just an old laptop available.